CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # These owners will be the default owners for everything in the repo.
-* @camunda/connectors
\ No newline at end of file
+* @camunda/connectors app/renovate-approve
```

Comment:
Is that possible?

https://github.com/orgs/community/discussions/23064

Reply:
I thought so! Others are using it:

https://github.com/search?q=app%2Frenovate-approve%20path%3A*CODEOWNERS&type=code

I'm looking into now why this isn't reflected yet in the PR.

Reply:
Ok, it looks like this isn't possible yet:

https://github.com/apps/renovate-approve

I'll follow up on Slack

Reply:
@sbuettner I excluded the files that Renovate often updates from CODEOWNERS.

While not perfect, this might be the best workaround to get most of the benefits of a protected branch rule, while also having some dependency upgrade automation.